e2e/tests/nightly/admin/institution-admin-ui.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { signInWithAccessToken } from 'utils/test-utils';
+import { config } from 'resources/workbench-config';
+import { HeaderName } from 'app/text-labels';
+import navigation, { NavLink } from 'app/component/navigation';
+import AdminTable from 'app/component/admin-table';
+import InstitutionAdminPage from 'app/page/admin-institution-list-page';
+import InstitutionEditPage, {
+  AcceptedAddressSelectValue,
+  InstitutionTypeSelectValue
+} from 'app/page/admin-institution-edit-page';
+
+describe('Institution Admin', () => {
+  const testInstitutionName = 'Admin testing';
+
+  beforeEach(async () => {
+    await signInWithAccessToken(page, config.ADMIN_TEST_ACCESS_TOKEN_FILE);
+    await navigation.navMenu(page, NavLink.INSTITUTION_ADMIN);
+  });
+
+  test('check the Institution Admin page UI and test-Institution edit page', async () => {
+    const institutionAdminPage = new InstitutionAdminPage(page);
+    await institutionAdminPage.waitForLoad();
+    const adminTable = new AdminTable(page);
```

Comment:
table loading takes some time. here you need to add `await adminTable.waitUntilVisible();`.